.travis.yml

```diff
@@ -1,7 +1,5 @@
 language: Java
 jdk:
   - oraclejdk8
-install: mvn -q clean -DskipTests -DskipITs install
-script: mvn -q clean verify
-after_success:
-  - mvn -q clean cobertura:cobertura-integration-test coveralls:report
+install: mvn -q clean -DskipTests -DskipITs install && mvn -q clean cobertura:cobertura-integration-test coveralls:report
```

Comment:
Ah, I was thinking "after_success" was more like "final" in bamboo, not "the next stage", like in maven.